src/core/Form/Checkbox/CheckboxGroup.tsx

```diff
@@ -74,11 +102,31 @@ class BaseCheckboxGroup extends Component<
               {labelText}
             </Label>
             {groupHintText && <HintText>{groupHintText}</HintText>}
+            {groupStatusText && (
+              <VisuallyHidden
+                {...getConditionalAriaProp('aria-labelledby', [statusTextId])}
```

Comment:
Is there a reason this is defined as an aria attribute instead of just content in the visually hidden? Both seem to work the same, just wondering.

Reply:
Tried to follow the pattern how **StatusText** was used elsewhere, so that it is referenced by it's **id**. 
So that the actual content is written only once, inside StatusText.

Reply:
Yeah, makes sense. Thanks for clarifying. With this approach I think the component it's defined in could be almost anything, but `VisuallyHidden` is probably the safest bet.

Reply:
True, it could be almost any element. I will leave it to use `VisuallyHidden` as it tells the purpose quite clearly.